superset/charts/schemas.py

```diff
@@ -705,6 +705,11 @@ class ChartDataExtrasSchema(Schema):
 
 
 class ChartDataQueryObjectSchema(Schema):
+    applied_time_extras = fields.Dict(
+        description="A mapping of temporal extras that have been applied to the query",
+        required=False,
+        example={"__time_range": "1 year ago : now"},
+    )
```

Comment:
I tried creating a dedicated schema with named properties (`__time_range` etc) and then adding a nested reference here, but that didn't work, probably due to the double underscores. Leaving these as a `fields.Dict` for now to get around the double underscore problem.